src/Interfaces/ISDK.ts

```diff
@@ -3,7 +3,7 @@ import IReconnectableChatsParams from "../Interfaces/IReconnectableChatsParams";
 import FetchChatTokenResponse from "../Model/FetchChatTokenResponse";
 
 export default interface ISDK {
-  getChatConfig(requestId: string): Promise<object>;
+  getChatConfig(requestId: string, bypassCache: boolean): Promise<object>;
```

Comment:
Should we default this to false so consumers of this don't have to update their code to explicitly specify false here?

Reply:
I believe that we are doing this in the SDK.ts and this is ok in the type definition
Should I add this to .d.ts?

Reply:
Yes, we don't want to change the interface as much as possible.

Reply:
Fixed